lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -2875,4 +2875,64 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
                 /*Torch_BoolType:$antialias*/ cstFalse);
         return success();
       });
+  patterns.onOp(
+      "Upsample", 9, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
```

Comment:
https://onnx.ai/onnx/operators/text_diff_Upsample_1_10.html

I'm not sure but it looks like this is general enough to support as early as Upsample 1.

Reply:
They expect the scales arg to be of different type.